Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM openjdk:8-jre-alpine
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.2
```

Comment:
can you use this image instead (it's the latest): `hmcts/cnp-java-base:openjdk-8u181-jre-alpine3.8-1.0`

Reply:
Thanks, done.